src/main/java/org/skriptlang/skript/lang/entry/SubContainerEntryData.java

```diff
@@ -0,0 +1,46 @@
+package org.skriptlang.skript.lang.entry;
+
+import ch.njol.skript.ScriptLoader;
+import ch.njol.skript.config.Node;
+import ch.njol.skript.config.SectionNode;
+import org.jetbrains.annotations.Nullable;
+import org.skriptlang.skript.lang.entry.EntryValidator.EntryValidatorBuilder;
+
+public class SubContainerEntryData extends EntryData<EntryContainer> {
```

Comment:
I think this could be renamed to be a bit clearer, like just being ContainerEntryData
Also needs a lot more javadocs